src/core/types.h

```diff
@@ -446,10 +446,12 @@ class Box {
 
     llvm::iterator_range<BoxIterator> pyElements();
 
+    size_t getHCAttrsOffset();
```

Comment:
It looks like we have a couple uses of attrs_offset that should be updated to call this instead -- the ones I see are in objmodel.cpp related to rewriting.  Could you take a look?  You can check out the test I added in #511